src/main/webapp/app/exercises/shared/dashboards/tutor/tutor-exercise-dashboard.component.ts

```diff
@@ -51,6 +52,8 @@ export class TutorExerciseDashboardComponent implements OnInit, AfterViewInit {
     exercise: Exercise;
     modelingExercise: ModelingExercise;
     courseId: number;
+    exam: Exam | null = null;
+    isExamMode = false;
```

Comment:
We are basically already holding this information in the `exam` variable, I'm not sure if it makes sense to make `isExamMode` a method that checks if `exam` is `null`.
But I believe thats a minor issue as the values are only set once if I see it correctly.